tests/end2end/features/misc.feature

```diff
@@ -441,6 +441,49 @@ Feature: Various utility commands.
         And I run :message-info {clipboard}bar{url}
         Then the message "{url}barhttp://localhost:*/hello.txt" should be shown
 
+    Scenario: Variable {url:pretty}
```

Comment:
Any chance you could take a quick look into what it'd take to make those unit rather than end2end tests? If it ends up not being possible easily then those are fine, but if it's possible, I'd strongly prefer that.

Reply:
I don't think I can make it unit tests (because most variable expansion requires a tabbedbrowser, which I'm not entirey sure I can properly mock), but I did get it to a python e2e test, does that sound ok?